go-controller/pkg/ovn/common.go

```diff
@@ -2,27 +2,14 @@ package ovn
 
 import (
 	"fmt"
-	"hash/fnv"
-	"strconv"
 
 	util "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
 	"k8s.io/klog/v2"
 )
 
-// hash the provided input to make it a valid addressSet or portGroup name.
```

Comment:
Is there any reason we don't move all these functions to the `util` package so that we can get rid of `common.go`? In terms of code readability I think it would make things easier

Reply:
Yeah, we probably should. But let's do that in another PR.